addons/mrp/models/mrp_production.py

```diff
@@ -1928,6 +1928,76 @@ def action_serial_mass_produce_wizard(self):
         }
         return action
 
+    def action_split_wizard(self):
+        if len(self) > 1:
+            if any(production.state not in ('draft', 'confirmed') for production in self):
+                raise ValidationError(_("Only manufacturing orders in either the draft or confirmed status can be splitted."))
+            productions = [Command.create({'production_id': production.id}) for production in self]
+            wizard = self.env['mrp.production.split'].create({'production_ids': productions})
+            action = self.env['ir.actions.actions']._for_xml_id('mrp.action_mrp_production_split')
+            action['res_id'] = wizard.id
+            return action
+        else:
+            if self.state not in ('draft', 'confirmed'):
+                raise ValidationError(_("Only manufacturing orders in either the draft or confirmed status can be splitted."))
+            wizard = self.env['mrp.production.split.production'].create({'production_id': self.id})
+            action = self.env['ir.actions.actions']._for_xml_id('mrp.action_mrp_production_split_production')
+            action['res_id'] = wizard.id
+            return action
+
+    def action_merge(self):
+        if len(self) == 1:
+            raise UserError(_("Nothing to Merge."))
+        if any(production.state not in ('draft', 'confirmed') for production in self):
+            raise ValidationError(_("Only manufacturing orders in either the draft or confirmed status can be merged."))
+        products = set([(production.product_id, production.bom_id) for production in self])
+        if len(products) > 1:
+            raise UserError(_('You can only merge manufacturing orders of identical products.'))
+        additional_raw_ids = self.mapped("move_raw_ids").filtered(lambda move: not move.bom_line_id)
+        additional_byproduct_ids = self.mapped('move_byproduct_ids')
```

Comment:
same check than upper. You can merge the byproduct if they come from the same BoM